easybuild/easyconfigs/m/monty/monty-0.6.4-intel-2015a-Python-2.7.9-gmatteo-20150325.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = "PythonPackage"
+
+name = 'monty'
+version = '0.6.4'
+
+homepage = 'http://pypi.python.org/pypi/monty/'
+description = """Monty implements supplementary useful functions for Python that are not part of the standard library.
+ Examples include useful utilities like transparent support for zipped files, useful design patterns such as singleton
+ and cached_class, and many more."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['https://github.com/gmatteo/monty/archive/']
+sources = ['3152d41.tar.gz']
```

Comment:
I'm using `--gmatteo-20150325` as a tag for this particular version/commit, including the commit ID in the versionsuffix is meaningless